admin/class-config.php

```diff
@@ -142,293 +142,4 @@ private function do_yoast_export() {
 
 		}
 	}
-
-	/********************** DEPRECATED METHODS **********************/
```

Comment:
I'm not sure if we want to remove this header as we might have deprecated methods in the feature.